metadata.yaml

```diff
@@ -27,6 +33,14 @@ requires:
   prometheus-target:
     interface: http
   prometheus-rules:
-    interface: prometheus-rules
+    interface: prometheus_rules
```

Comment:
I do not think this is correct, see https://git.launchpad.net/charm-prometheus2/tree/src/metadata.yaml#n40